src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -203,6 +210,22 @@ private ChromeOptions buildChromeOptions(TestName testName) throws IOException {
         return chromeOptions;
     }
 
+    private SafariOptions buildSafariOptions(TestName testName) throws IOException {
```

Comment:
untested, I have no mac. 
noticed that we don't set any custom things for safari.